src/js/components/Main/MainComponents/Navbar/Navbar.jsx

```diff
@@ -1,15 +1,43 @@
 import React from 'react'
-import {NavLink} from "react-router-dom";
-
-function Navbar(props) {
-    return(<div>
-        <NavLink to="/admin">admin</NavLink><br/>
-        <NavLink to="/login">Login</NavLink><br/>
-        <NavLink to="/">home</NavLink><br/>
-        <NavLink to="/search">search</NavLink><br/>
-        <NavLink to="/schedule">schedule</NavLink><br/>
-        <NavLink to="/contacts">contacts</NavLink><br/>
-    </div>)
+import {NavLink, useLocation } from "react-router-dom";
+
+import {Navbar, Nav} from "react-bootstrap";
+
+import style from './Navbar.module.css'
+
+import "./Navbar.css"
+
+
+function NavbarF(props) {
```

Comment:
Why `NavbarF`? Is this a typo?

Reply:
That was template editing, that meant NavbarFunction. But now it is useless and typical Navbar will correct.

Reply:
Rename it then